src/main/java/teammates/logic/api/Logic.java

```diff
@@ -301,6 +301,15 @@ public void putDocument(InstructorAttributes instructor) {
     }
 
     /**
+     * Batch creates or updates documents for the given Instructors.
+     *
+     * @see InstructorsLogic#putDocuments(List)
+     */
+    public void batchPutInstructorDocuments(List<InstructorAttributes> instructors) {
```

Comment:
hmm.. but if i rename it this way, I end up with `instructorsLogic.putInstructorDocuments(instructors);` the extra `Instructor` feels redundant here. Would there be a better way to name this?